test/nextjs.test.js

```diff
@@ -729,58 +740,30 @@ if (major >= 18) {
 
             const authenticatedRequest = new NextRequest("http://localhost:3000/api/get-user", {
                 headers: {
-                    Authorization: `Bearer ${tokens.access}`,
+                    Cookie: `sAccessToken=${tokens.access}`,
                 },
```

Comment:
pls check if any of the examples in this repo also need changing. The nextjs one will need for sure since we no longer pass NextResponse to getAppDirRequestHandler

Reply:
I already updated the next example and create-supertokens-app as well as shared a video of testing with the updated code.